examples/experimental/dagster-airlift/dagster_airlift_tests/unit_tests/conftest.py

```diff
@@ -122,3 +129,28 @@ def assert_dependency_structure_in_assets(
         assert {dep.asset_key for dep in spec.deps} == {
             AssetKey.from_user_string(dep) for dep in deps_list
         }
+
+
+class DefinitionsLoadContextFake(DefinitionsLoadContext):
+    def __init__(
+        self,
+        metadata: Mapping[str, Any] = {},
+        load_type: DefinitionsLoadType = DefinitionsLoadType.INITIALIZATION,
+    ):
+        super().__init__(load_type=load_type)
+        self._metadata = metadata
+
+    @property
+    def reconstruction_metadata(self) -> Mapping[str, Any]:
+        return self._metadata
```

Comment:
You shouldn't need this. You can construct a `DefinitionsLoadContext` directly and substitute the right values.

Reply:
DefinitionsLoadContext requires a repository load data be passed in, which is quite tedious to construct. I found this easier

Reply:
`RepositoryLoadData({}, metadata)`

?